src/Callback.php

```diff
@@ -53,7 +53,7 @@ public function getUrlTrigger(): string
     /**
      * Executes user-specified action when call-back is triggered.
      *
-     * @param \Closure $fx
+     * @param callable $fx
```

Comment:
we should keep `\Closure` to be stricter and prevent contextual issues with generic callable